src/components/ImageUpload.css

```diff
@@ -1,16 +1,42 @@
 @import '../variables.css';
 
 ._legoEditor_imageUploader_dropZone {
-  width: 80%;
+  display: flex;
+  width: 325px;
```

Comment:
Could this also get a max-width of somewhere along 90-100% to be sure that it will not overflow on smaller screens?

Reply:
Sure - I can look into it